modules/oe_theme_helper/tests/src/Kernel/Plugin/Field/FieldFormatter/FeaturedMediaFormatterTest.php

```diff
@@ -0,0 +1,192 @@
+    $image = File::create(['uri' => 'public://example.jpg']);
+    $image->save();
+
+    $media = \Drupal::service('entity_type.manager')
+      ->getStorage('media')->create([
+        'bundle' => 'image',
+        'name' => 'Test image',
+        'oe_media_image' => [
+          [
+            'target_id' => $image->id(),
+            'alt' => 'default alt',
+          ],
+        ],
+      ]);
+    $media->save();
+
+    $values = [
+      'type' => 'test_ct',
+      'title' => 'My node title',
+      'featured_media_field' => [
+        [
+          'target_id' => $media->id(),
+          'caption' => 'Image caption text',
+        ],
+      ],
+    ];
+
+    /** @var \Drupal\node\NodeInterface $node */
+    $node = Node::create($values);
+    $node->save();
+
+    $node_storage = \Drupal::service('entity_type.manager')->getStorage('node');
+    $node_storage->resetCache();
+
+    /** @var \Drupal\node\NodeInterface $node */
+    $node = $node_storage->load($node->id());
+
+    $view_builder = \Drupal::service('entity_type.manager')->getViewBuilder('node');
+
+    $build = $view_builder->viewField($node->get('featured_media_field'), [
+      'type' => 'oe_theme_helper_featured_media_formatter',
+    ]);
+
+    $this->assertRendering($this->renderRoot($build), [
+      'count' => [
+        '.ecl-media-container .ecl-media-container__media' => 1,
+        '.ecl-media-container .ecl-media-container__caption' => 1,
+      ],
+      'equals' => [
+        '.ecl-media-container__caption' => 'Image caption text',
+      ],
+      'contains' => [
+        'default alt',
+        'example.jpg',
+      ],
+    ]);
+
+    $media = \Drupal::service('entity_type.manager')
+      ->getStorage('media')->create([
+        'bundle' => 'remote_video',
+        'oe_media_oembed_video' => 'https://www.youtube.com/watch?v=OkPW9mK5Vw8',
+        'status' => 1,
+      ]);
+    $media->save();
+
+    $values = [
+      'type' => 'test_ct',
+      'title' => 'My video node',
+      'featured_media_field' => [
+        [
+          'target_id' => $media->id(),
+          'caption' => 'Video caption text',
+        ],
+      ],
+    ];
+
+    $node = Node::create($values);
+    $node->save();
+
+    $node_storage->resetCache();
+    $node = $node_storage->load($node->id());
+
+    $build = $view_builder->viewField($node->get('featured_media_field'), [
+      'type' => 'oe_theme_helper_featured_media_formatter',
+    ]);
+
+    $this->assertRendering($this->renderRoot($build), [
+      'count' => [
+        '.ecl-media-container .ecl-media-container__media' => 1,
+        '.ecl-media-container iframe.media-oembed-content' => 1,
+        '.ecl-media-container .ecl-media-container__caption' => 1,
+      ],
+      'equals' => [
+        '.ecl-media-container__caption' => 'Video caption text',
+      ],
+      'contains' => [
+        UrlHelper::encodePath('https://www.youtube.com/watch?v=OkPW9mK5Vw8'),
```

Comment:
Same as above, let's move it under "count" and use something like the following:
```
 'div.ecl-hero-banner__image[style="background-image:url(http://via.placeholder.com/150x150)"]': 1
```

Reply:
Fixed